tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -246,6 +246,10 @@ def quantile_levels(self):
     @property
     def eval_metric(self):
         return self._learner.eval_metric
+    
+    @property
+    def original_features(self):
+        return self._learner.original_features
```

Comment:
Add documentation:

 """Original features user passed in to fit before processing"""